packages/shared/src/BootcAPI.ts

```diff
@@ -33,6 +36,8 @@ export abstract class BootcApi {
   abstract deleteBuilds(buildIds: string[]): Promise<void>;
   abstract selectOutputFolder(): Promise<string>;
   abstract selectBuildConfigFile(): Promise<string>;
+  abstract selectVMImageFile(): Promise<string>;
```

Comment:
I'd really like to simplify this and not keep creating select*File functions - but totally fine to follow the pattern for this PR. 👍🏼